src/accelerate/inference.py

```diff
@@ -111,9 +114,11 @@ def prepare_pippy(
     example_args=(),
     example_kwargs: Optional[Dict[str, Any]] = None,
     num_chunks=None,
+    send_output_to_cpu=True,
```

Comment:
it should be false no according to the docstring. Let's add the description of this arg.